notebooks/Manage Map Areas with Group and Index.ipynb

```diff
@@ -0,0 +1,456 @@
+    "2. Map areas will be created as defined in hosted feature layer: `Manage Map Area Index`\n",
+    "    - Known limitations:\n",
+    "        - no more than 16 map areas can be defined for the web map\n",
+    "        - a single tile/vector package or local db cannot exceed 4 GB\n",
+    "    - The following map area processing and metadata details are defined in `Manage Map Area Index`\n",
+    "        - title\n",
+    "        - snippit \n",
+    "        - tags\n",
+    "        - refresh schedule details\n",
+    "        - folder \n",
+    "    - Process results will be added to a process log table: `Manage Map Area Processing Log`\n",
+    "    - Reference: https://developers.arcgis.com/python/latest/api-reference/arcgis.map.toc.html#offlinemapareamanager\n",
+    "3. A summary report is emailed to members of `Manage Map Area` group\n",
+    "    - Reference: https://community.esri.com/t5/arcgis-notebooks-documents/send-e-mail-notifications-with-arcgis-notebooks/ta-p/1329699\n",
+    "\n",
+    "---\n",
+    "\n",
+    "#### Notes on map area download / syncing:\n",
+    "- When a map area is created, a copy of the data is packaged and made available for mobile users to download. This serves as the initial dataset\n",
+    "- The frequency at which the initial dataset refreshes can be defined during the creation of the map area. Please ensure this is set according to the needs of the field operations.\n",
+    "- When field workers download an map area, they store that copy locally on their devices. Any subsequent edits made to the layer's data will be synced back to portal.\n",
+    "- In some circumstances field workers will need to remove and redownload map areas to ensure they have latest changes. \n",
+    "    - Map areas recreated to include changes to schema changes, field domain changes, new layers\n",
+    "    - Change to web map (form, popup, symbology) \n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Import ArcGIS Libraries and Connect to GIS"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from arcgis.gis import GIS\n",
+    "from arcgis.map import Map # new to 2.4\n",
+    "from arcgis.layers import VectorTileLayer\n",
+    "from arcgis.layers import MapImageLayer\n",
+    "import datetime, time\n",
+    "#from arcgis import env # << uncomment both lines for additional logging detail    \n",
+    "#env.verbose = True\n",
+    "\n",
+    "gis = GIS(\"home\") "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Get Manage Map Areas HFL and Group \n",
+    "\n",
+    "See `Setup` notes above and copy/paste in item ids were shown below"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# ACTION REQUIRED: see SETUP notes above\n",
```

Comment:
Nice, helpful for the user to know they should change this value. We could do something like:

`MAP_AREAS_HFL_ITEM_ID = input()` and `MAP_AREAS_GROUP_ITEM_ID = input()` with some print statements above each so that when the user runs this cell they are prompted to put in their item id?

Reply:
Once the user copy/pastes in the item ids and saves the notebook they won't have to update in the future. It's a one time config.  The input() method would ask for input each time they run.

The file gdb does not have to be in the group.  Doesn't hurt anything.  The process excludes items in group that are not web maps